ibm-cloud-pak-for-data/additionalnodes.json

```diff
@@ -29,43 +29,36 @@
 			}
 		},
 		"nodeOrMaster": {
-            "type": "string",
+			"type": "string",
```

Comment:
let's remove this file for now - we'll add it as another sample once this is merged.

Reply:
Done.